exampleCourse/questions/demoRandomPlot/question.html

```diff
@@ -1,5 +1,6 @@
 <pl-question-panel>
-    <pl-figure file-name="figure.png" type="dynamic" width="500px"></pl-figure>
+    <!-- "my-mpl-figure" here is not a physical file. Since this pl-figure is the "dynamic" type, the figure is generated via server.py's `file()`. -->
+    <pl-figure file-name="my-mpl-figure" type="dynamic" width="500px"></pl-figure>
```

Comment:
When I first tried to figure out what was going on here, I couldn't figure out where `figure.png` was. I kept thinking "both `question.html` and `server.py` keep referencing this `figure.png`, but I can't find it anywhere." Mariana ended up explaining how it worked in OH.

I think it's easier for new users to understand if the dynamic figure `file-name` does not look like a physical file, but I can revert this you think commenting is sufficient.

Reply:
I definitely like the comment. One reason to prefer a filename that "looks real" is for use with `<pl-download>`, where a generated PDF file should be saved with filename extension `.pdf`, for example. I guess to me it makes sense to always use "real looking" filenames, but I acknowledge that dynamic files are just inherently tricky.

Maybe it's worth adding a special section on dynamic files, where you really clearly explain that the file never exists as a file on disk in the conventional sense, and that it's all "fake"?

Reply:
Added a `question.md` subsection and slightly modified the `pl-figure` docs. Not sure if that's what you had in mind.

Reply:
Looks great!